services/movie_session.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+
+from django.core.exceptions import ObjectDoesNotExist
+from django.db.models import QuerySet
+
+from db.models import MovieSession, Movie, CinemaHall
+from datetime import datetime
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    try:
+        movie = Movie.objects.get(id=movie_id)
+        cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    except ObjectDoesNotExist:
+        raise ValueError("Provided movie or cinema hall ID does not exist.")
```

Comment:
Using `ObjectDoesNotExist` is too broad and may catch exceptions from unrelated queries. Consider using `Movie.DoesNotExist` and `CinemaHall.DoesNotExist` separately for more precise exception handling.